src/Materials/standardMaterial.ts

```diff
@@ -1229,7 +1229,7 @@ export class StandardMaterial extends PushMaterial {
             DetailMapConfiguration.AddSamplers(samplers);
 
             PrePassConfiguration.AddUniforms(uniforms);
-            PrePassConfiguration.AddSamplers(uniforms);
+            PrePassConfiguration.AddSamplers(samplers);

```

Comment:
good catch!

Reply:
No impact currently because `PrePassConfiguration.AddSamplers` has an empty implementation, but who knows in the future!